app/models/concerns/has_commands.rb

```diff
@@ -0,0 +1,36 @@
+module HasCommands
```

Comment:
Add a comment that it expects `command_associations` to be defined by including class.